eclair-node/src/main/scala/fr/acinq/eclair/api/handlers/PathFinding.scala

```diff
@@ -29,26 +29,26 @@ trait PathFinding {
   import fr.acinq.eclair.api.serde.JsonSupport.{formats, marshaller, serialization}
 
   val findRoute: Route = postRequest("findroute") { implicit t =>
-    formFields(invoiceFormParam, amountMsatFormParam.?) {
-      case (invoice@PaymentRequest(_, Some(amount), _, nodeId, _, _), None) =>
-        complete(eclairApi.findRoute(nodeId, amount, invoice.routingInfo))
-      case (invoice, Some(overrideAmount)) =>
-        complete(eclairApi.findRoute(invoice.nodeId, overrideAmount, invoice.routingInfo))
+    formFields(invoiceFormParam, amountMsatFormParam.?, "experimentName".as[String].?) {
```

Comment:
It's probably worth adding a couple tests to `ApiServiceTests` just to check that we correctly forward the given `experimentName` to the eclair instance?

Reply:
Added some tests